src/frontend/src/lib/util/util.ts

```diff
@@ -66,17 +76,42 @@ export const mod_panel_permissions = [
 
 export const valid_games: { [key: string]: string } = {
   mk8dx: 'Mario Kart 8 Deluxe',
+  mk8: 'Mario Kart 8',
   mkw: 'Mario Kart Wii',
   mkt: 'Mario Kart Tour',
+  mk7: 'Mario Kart 7',
+  smk: 'Super Mario Kart',
 };
 export const valid_modes: { [key: string]: string[] } = {
-  mk8dx: ['150cc', '200cc'],
+  mk8dx: [
+    '150cc',
+    '200cc',
+    'mixed_battle',
+    'ballon_battle',
+    'shine_thief',
+    'bobomb_blast',
+    'coin_runners',
+    'renegade_roundup',
+    'match_race',
+    'mixed',
+  ],
+  mk8: ['150cc', '200cc'],
   mkw: ['rt', 'ct'],
   mkt: ['vsrace'],
+  mk7: ['vsrace'],
+  smk: ['match_race'],
```

Comment:
Make sure that anything here also has a counterpart in `/backend/common/data/models/common.py` in the `Game` and `GameMode` literals, from what I can tell they haven't been changed in this PR so that needs to be done